src/types.ts

```diff
@@ -21,11 +21,70 @@ interface IAnswerInfo {
   text: string;
 }
 
+interface IQuizSetupProps {
```

Comment:
We don't need to prefix types or interfaces with `I`. It was a popular convention in the past, but recently, developers have started dropping the `I` prefix. Then again it comes to personal preference so not a biggy

Reply:
I like this convention so I think I'll keep it for now